src/Security/Group.php

```diff
@@ -105,6 +105,16 @@ public function getAllChildren()
 
         return $doSet;
     }
+    
+    private function getDecodedBreadcrumbs()
+    {
+        $list = Group::get()->exclude('ID', $this->ID);
+        $groups = ArrayList::create();
+        foreach ($list as $group) {
+            $groups->push(['ID' => $group->ID, 'Title' => Convert::xml2raw($group->Breadcrumbs)]);
```

Comment:
This will work better.
```suggestion
            $groups->push(['ID' => $group->ID, 'Title' => $group->getBreadcrumbs(' » ')]);
```
By using `Convert::xml2raw` you risk converting html entities that should be converted. E.g.: Let's say you create a group called `My group &eacute;` ... in the drop down, it will be rendered as `My group é`